### PR TITLE
Add maximum growth

### PR DESCRIPTION
**Pull request by ravi1001**
Please review this, @casey-collab

[Review on Reviewable](https://reviewable.io/reviews/udacity/create-your-own-adventure/1611)

### CODE REVIEW

**Comment by cbuckey-uda** [comment]
@casey-collab, you there?

File: english/max-growth/max-growth.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Your maximum growth occurs at the point of your maximum resistance.
```
Comment (casey-collab) [inline english/max-growth/max-growth.md]: It looks like this page is not reachable from the rest of the story. Make you sure you add a link to an existing page that leads to this page, or readers will not be able to find your addition.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**